examples/local_pdc_dev.rb

```diff
@@ -9,16 +9,19 @@ def main
   end
 
   releases = PDC::V1::Release.all!.to_a
-  ap releases
-
   released_files = PDC::V1::ReleasedFile.all!.to_a
+  ap releases
```

Comment:
This change is not relevant to belongs_to.

Reply:
Aha, the latest change was merged in another patch, after roll back the previous changes in this pr, the codes came to old one, will change back.